### DISCUSSION TITLE
LocalSend Android no detecta pc con windows 10

### DISCUSSION THREAD

**Discussion by Troyanwolf** [localsend/localsend#1404] [L-Español]
Buenas, he instalado la aplicacion localsend tanto en mi samsung galaxi a03s (andorid )como asi tambien en mi pc con windows 10, el tema es que desde el movil no me detecta la pc, sin embargo desde la pc si que me detecta el movil y puedo enviar archivos. 
Hay que realizar alguna configuracion extra para que funcione  o no. Cualquier ayuda es bienvenida. Desde ya gcs.

**Comment by jsmsalinas** [comment]
Yo tengo mi duda también..

Por qué sale debajo del nombre "alias", tres números con hashtag tipo #72 #0 #59 y cuando se envía solo deja poner "#72"? Quiere eso decir que el archivo se envía a todos los que tienen el #72? Y de no ser así, por qué salen tres números hashtag si solo deja poner uno?

**Reply by Caltreme** [reply]
El problema de envio entre pc y android es por el firewall, tienes que configurarlo para que permita recibir archivos de localsend. Yo tenia el mismo problema con linux. Aquí la liga donde hablan del tema y la solución, el tema es sobre linux, pero mas abajo hay de windows.
https://github.com/localsend/localsend/discussions/1075
